src/main/java/org/terasology/tooltip/TooltipClientSystem.java

```diff
@@ -30,8 +31,9 @@
 
 @RegisterSystem(RegisterMode.CLIENT)
 public class TooltipClientSystem extends BaseComponentSystem {
-    @ReceiveEvent(components = OreonAttributeComponent.class)
-    public void addAttributesToTooltip(GetItemTooltip event, EntityRef entity, OreonAttributeComponent oreonAttributeComponent) {
+    @ReceiveEvent(components = {OreonAttributeComponent.class, CreatureNameComponent.class})
+    public void addAttributesToTooltip(GetItemTooltip event, EntityRef entity, OreonAttributeComponent oreonAttributeComponent,  CreatureNameComponent oreonNameComponent) {
+        event.getTooltipLines().add(new TooltipLine("Name : " + oreonNameComponent.firstName + " " + oreonNameComponent.lastName));
```

Comment:
These attribute names need to be localized / internationalized / translated.
I'd recommend using constants for these.  I don't recall how translation works for Terasology off the top of my head.
There's probably past non-log display areas where you should also be doing translations.